crates/derive/src/stages/l1_traversal.rs

```diff
@@ -108,8 +108,21 @@ impl<F: ChainProvider + Send> OriginAdvancer for L1Traversal<F> {
         }
 
         crate::set!(ORIGIN_GAUGE, next_l1_origin.number as i64);
+
+        let prev_block_holocene =
+            self.rollup_config.is_holocene_active(self.block.unwrap().timestamp);
```

Comment:
This is safe, but can we pls add a comment for why we can `unwrap` here (or just use `expect` w/ a message that it's infallible due to the first check.)